app/app.go

```diff
@@ -727,6 +727,15 @@ func New(
 		return updatedVM, nil
 	})
 
+	// this upgrade is for breaking bug fixes on testnet
```

Comment:
it's necessary for current testnet nodes(on 0.7.0-rc3) to use recent 0.7.0 binaries.